### DISCUSSION TITLE
allure-jenkins-plugin is not working

### DISCUSSION THREAD

**Discussion by skamineni** [allure-framework/allure2#1330] [Bug Reports] [answered]
Forbidden: Failed to download https://dl.bintray.com/qameta/generic/io/qameta/allure/allure/2.7.0/allure-2.7.0.zip 

Can somone please provide suggestions on how to make this work to publish allure report in Jenkins

**Comment by cheshi-mantu** [comment]
@skamineni Hi Lakshmi,
could you please provide a bit more information about your issue? Like what are the versions of components you are using etc.

**Comment by skamineni** [comment, marked as answer]
We are all set here. We have changed the plugin version to use 2.13 instead of 2.7. Thanks!

From: Egor Ivanov ***@***.***>
Reply-To: allure-framework/allure2 ***@***.***>
Date: Monday, June 28, 2021 at 10:24 AM
To: allure-framework/allure2 ***@***.***>
Cc: Lakshmi Kamineni-Quickbase ***@***.***>, Mention ***@***.***>
Subject: Re: [allure-framework/allure2] allure-jenkins-plugin is not working (#1330)


@skamineni<https://github.com/skamineni> Hi Lakshmi,
could you please provide a bit more information about your issue? Like what are the versions of components you are using etc.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/allure-framework/allure2/discussions/1330#discussioncomment-933122>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AEKZNEGNLPV5MTXXBE46K4LTVCA2LANCNFSM47FTVZMA>.

This message and its contents are confidential. If you received this message in error, do not use or rely upon it. Instead, please inform the sender and then delete it. Thank you.
